doc/nrf/ug_thingy53.rst

```diff
@@ -219,60 +189,158 @@ You can program the precompiled firmware image using one of the following ways:
+The Nordic Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with the Nordic Thingy:53 by default include the MCUboot bootloader with serial recovery support.
+You can update applications compatible with the Nordic Thingy:53 using the DFU functionality with either `nRF Connect Programmer`_ or the `nRF Programmer`_ mobile application.
 
 Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
-For a sample or application to be compatible with Thingy:53, the application must comply with the following configuration:
+For a sample or application to be compatible with Thingy:53, the application must comply with the configurations described in the following sections.
+
+.. note::
+   To use your application with the Nordic Thingy:53 preprogrammed bootloader, you must only set the proper configuration for the Partition Manager and MCUboot.
+   Configurations that enable the other features are not mandatory, but you can enable them if needed.
+
+   All applications compatible with the Nordic Thingy:53 in the |NCS| also support the additional features.
+   Refer to the source code of the samples and applications for examples of implementation.
+
+.. _thingy53_app_partition_manager_config:
 
 Partition manager configuration
-  Thingy:53 compatible samples use :ref:`partition_manager` to define memory partitions.
-  To ensure that partition layout does not change between builds, the sample must use static partition layout that is the same as the one in Thingy:53-compatible samples in the |NCS|.
-  The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+===============================
+
+The samples compatible with the Nordic Thingy:53 use :ref:`partition_manager` to define memory partitions.
+To ensure that partition layout does not change between builds, the sample must use a static partition layout that is consistent all compatible samples in the |NCS|.
+See :file:`pm_static_thingy53_nrf5340_cpuapp.yml` in a compatible sample for the configuration used.
+The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+
+The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
+Trying to access data on this partition results in an ARM fault.
 
-  The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
-  Trying to access data on this partition results in an ARM fault.
+You must add a flash controller overlay for the application core to allow the MCUboot bootloader load data from the external flash memory to the network core using the application core RAM.
```

Comment:
You are mixing 2 things here:
The overlay is added to both MCUboot and to the application firmware.

1. MCUboot
The overlay with flash sim is added automatically in the background by the `nrf/modules/mcuboot/CMakeLists.txt`.  The overlay is needed for the described net core image updated.
2. Application
The overlay is added by the `thingy53_nrf5340_cpuapp.overlay` file from app configuration directory, but for different reason:
`The partition must be enabled by the application configuration to prevent bus faults when mcumgr tries to access simulated partition with network core image data.`

Reply:
Can we align it a bit?

Reply:
```suggestion
Both the MCUboot bootloader and the application core need a flash controller overlay.
For the MCUboot bootloader this is needed for the network core image update, and the overlay is automatically applied.
For the application core this is needed to prevent bus faults when MCUmanager tries to access the simulated partition that contains the network core image data.
See :file:`thingy53_nrf5340_cpuapp.overlay` in a compatible sample for how this is done for the application core.
```
Would this work as a replacement of the current Lines 226 - 228?

Reply:
I think so. Thanks

Reply:
`Both the MCUboot bootloader and the application need a flash controller overlay.`

Reply:
The MCUboot bootloader and user application both run on the application core ;)

Reply:
Right. Changed all references from "application core" to "application" in this bit.